services/webinar/api.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import { Headers } from 'node-fetch';
+
+import { fetchWithBody, apiBase } from './utils';
+
+export function registerParticipant({
+  firstName,
+  lastName,
+  email,
+  organizerKey,
+  webinarKey,
+  accessToken,
+}: {
+  firstName: string,
+  lastName: string,
+  email: string,
+  organizerKey: string,
+  webinarKey: string,
+  accessToken: string,
+}) {
+  const url = `${apiBase}/G2W/rest/organizers/${organizerKey}/webinars/${webinarKey}/registrants`;
+  const query = '?resendConfirmation=true';
+
+  const headers = new Headers();
+  headers.append('Accept', 'application/vnd.citrix.g2wapi-v1.1+json');
```

Comment:
What is the 'application/vnd.citrix.g2wapi-v1.1+json' bit for?

Reply:
Needed to tell GoToWebinar to use version 1.1 of the API. Described here: https://goto-developer.logmeininc.com/content/gotowebinar-api-reference